### PR TITLE
Add serde

### PR DESCRIPTION
**Pull request by kellpossible**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by kellpossible** [comment]
@conda-forge/help-python this is ready for review I think

File: recipes/serde/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - isodate >=0.6
```
Comment (dopplershift) [inline recipes/serde/meta.yaml:23]: [This version](https://github.com/rossmacarthur/serde/blob/71662749f5eca79e801df6642987cfc912c1c661/setup.py#L46) has a dependency on `six` as wel;:
```suggestion
    - isodate >=0.6
    - six >=1.13.0
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just a minor tweak and this is good to go.

File: recipes/serde/meta.yaml (hunk 1 above)
Comment (kellpossible) [reply recipes/serde/meta.yaml:23]: great thanks!

**Comment by kellpossible** [comment]
@dopplershift 

> Just a minor tweak and this is good to go.

I've added the dependency on `six` so hopefully it's good to go pending checks!

**Review by dopplershift — APPROVED** [review]

**Pull request merged**